src/Core/Console/AutomaticInstallation.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace Friendica\Core\Console;
+
+use Asika\SimpleConsole\Console;
+use dba;
+use Friendica\App;
+
+require_once 'mod/install.php';
+require_once 'include/dba.php';
+
+class AutomaticInstallation extends Console
+{
+	protected function getHelp()
+	{
+		return <<<HELP
+Installation - Install Friendica automatically
+Synopsis
+	bin/console install [-h|--help|-?] [-v] [-a] 
+
+Description
+	bin/console install
+		Installs Friendica with data based on the htconfig.php file
+
+Notes:
+    Not checking .htaccess/URL-Rewrite during CLI installation.
+
+Options
+    -h|--help|-? Show help information
+    -v           Show more debug information.
+    -a           All setup checks are required (except .htaccess)
+HELP;
+	}
+
+	protected function doExecute()
+	{
+		// remove die and copy config file
+		$fileContent = file_get_contents('./htconfig.php');
+		$fileContent = str_replace('die', '//die', $fileContent);
+		file_put_contents('.htautoinstall.php', $fileContent);
```

Comment:
`htconfig.php` contains default values that will certainly make this command fail if the `htconfig.php` hasn't been modified prior to its usage. This is the meaning of the `die()` line. If an external command changes the `htconfig.php` prior to its use by this command, I think it should remove the `die()` command at this stage, and not during this command.

And if an external script is modifying the `htconfig.php` file, then surely it can create the expected `.htautoinstall.php` or even a barebone `.htconfig.php`.

Reply:
This code has been removed.